index.js

```diff
@@ -358,15 +343,17 @@ LiveServer.start = function(options) {
 		ignoreInitial: true
 	});
 	function handleChange(changePath) {
-		var cssChange = path.extname(changePath) === ".css" && !noCssInject;
+		var cssChange = path.extname(changePath) === ".css";
 		if (LiveServer.logLevel >= 1) {
 			if (cssChange)
 				console.log("CSS change detected".magenta, changePath);
 			else console.log("Change detected".cyan, changePath);
 		}
 		clients.forEach(function(ws) {
-			if (ws)
-				ws.send(cssChange ? 'refreshcss' : 'reload');
+			if (ws) {
+				var file = changePath.replace(process.cwd()+'/', '');
```

Comment:
Instead of `changePath.replace(process.cwd()+'/', '')` use `path.basename('/foo/bar/baz/asdf/quux.html');`